### PR TITLE
cc3200: appsign.sh: Use md5 if running under Darwin

### PR DESCRIPTION
**Pull request by garyaj**

### CODE REVIEW

File: cc3200/appsign.sh [hunk 1]
```diff
@@ -13,7 +13,12 @@ BTYPE=$2
 BUILD=build/${BOARD}/${BTYPE}
 
 # Generate the MD5 hash
+# md5 on Darwin, md5sum on Unix
+if [ `uname -s` = "Darwin" ]; then
+echo -n `md5 $BUILD/application.bin | awk '{ print $4 }'` > __md5hash.bin
```
Comment (Anton-2) [inline cc3200/appsign.sh]: You can use `md5 -q $BUILD/application.bin` to get only the checksum, without using awk

Comment (garyaj) [inline cc3200/appsign.sh]: Thanks for that. I was so focussed on simply making it run on my Mac I
didn't look any further.

Rgds

Gary

**Comment by danicampora** [comment]
Thanks, looks OK.

**Comment by danicampora** [comment]
@garyaj please merge the commits into one, and push again with --force.

**Comment by danicampora** [comment]
Merged in https://github.com/micropython/micropython/commit/7080e9632c98b06952ee480f4b4d0021e43d63b4, thanks!

**Pull request closed without merging**